<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFourOutputInfo</class>
 <widget class="QDialog" name="CFourOutputInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DYNAMICALLY CHANGED</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>133</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QCheckBox" name="numDataChkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>17</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numerical Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="pdfCBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>109</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Output to pdf</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>147</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>100</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QCheckBox" name="amplChkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amplitude Chart</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="phaseChkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Phase Chart</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="bwCBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>125</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Black and white</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
